scripts/InitDatabase.sql

```diff
@@ -13,3 +14,11 @@ CREATE TABLE Dimensions(
   name TEXT,
   value TEXT
 );
+
+CREATE TABLE Downloads(
+  id SERIAL PRIMARY KEY,
```

Comment:
No it's not known or used for anything via API, could be said the same for Dimension table and hence the filter-api can set these.